src/components/SearchCard/SearchCard.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from './SearchCard.module.css';
+import { SearchCardProps } from '../../lib/types';
+
+export default function SearchCard({ searchResults }: SearchCardProps) {
+  return (
+    <>
+      {searchResults.map((searchCard, index) => (
+        <article className={styles.searchCard} key={index}>
```

Comment:
You shouldn't use the index as key. You can find an explanation [HERE](https://reactjs.org/docs/lists-and-keys.html#:~:text=We%20don't%20recommend%20using,an%20index%20as%20a%20key.). 
You could use the ID as key for example instead.